src/model/channel/sticker.rs

```diff
@@ -0,0 +1,57 @@
+use crate::model::id::{StickerId, StickerPackId};
+
+/// A sticker sent with a message
+///
+/// Bots currently can only receive messages with stickers, not send
```

Comment:
```suggestion
/// Bots currently can only receive messages with stickers, not send.
```
Adding another fullstop.